### ISSUE TITLE
Draw the whole fuzzy subcontroller 

### ISSUE THREAD

**Issue by JJ** [salem23/ConfofGames2019#28]
Clarifying the values of all the variables. For instance, it's not clear to me why there are only two values for every MF. Shouldn't there be 4? What are the values for x1 and x4? Are they standard and always the same for every MF depending on its linguistic variable, like x1 = 0 for the first, 0.25 for the second and so on? I don't think we have written that in any of our papers, so it would be great to show it for the first time in a clarifying figure.
We should also represent the evolved controllers in the last part of the paper, showing the actual MFs for every one of the three controllers, as well as explain the rules that actually drive the car.

**Comment by JJ** [comment]
I see the values were described in the second paper and never again... We probably need a table too.

**Comment by amorag** [comment]
JJ, it was explained in CoG 2019 paper also. There were several equations justifying why we just need two values per MF (6 per variable).
We removed it from this work.

Moreover, the variables (FRONT, M5 and M10) were defined for the subcontrollers, as they are used in the conditions of the fuzzy rules.
We could include these rules again in the text (they were removed also) or show them in a table, if it is clearer, but anyway, I would do it in the sub-controllers section, rather than in the TORCS section.
